src/test/kotlin/no/digdir/fdk/searchservice/integration/FilterTest.kt

```diff
@@ -476,4 +477,27 @@ class FilterTest: ApiTestContext() {
             Assertions.assertEquals(0, result.size)
         }
     }
+
+    @Nested
+    inner class Sorting {
+        @Test
+        fun `sorting on descending firstHarvested returns correct order`() {
+            val searchBody = mapper.writeValueAsString(
+                SearchOperation(
+                sort = SortField(
+                    field = SortFieldEnum.FIRST_HARVESTED, direction = SortDirection.DESC)
+                )
+            )
+
+            val response = requestApi(ALL_RESOURCES_PATH, port, searchBody, HttpMethod.POST)
+            Assertions.assertEquals(200, response["status"])
+
+            val result: List<SearchObject> = mapper.readValue(response["body"] as String)
+            Assertions.assertNotEquals(0, result.size)
```

Comment:
For å vise at testen ikke bare sorterer en liste med 1 innslag
```suggestion
            Assertions.assertTrue(1 < result.size)
```